cpp/benchmarks/fixture/nvbench_fixture.hpp

```diff
@@ -75,23 +82,64 @@ struct nvbench_base_fixture {
               "\nExpecting: cuda, pool, async, arena, managed, or managed_pool");
   }
 
+  inline auto make_cuio_host_pinned()
+  {
+    static std::shared_ptr<rmm::mr::pinned_memory_resource> mr =
+      std::make_shared<rmm::mr::pinned_memory_resource>();
+    return host_resource_ref(*mr);
+  }
+
+  inline auto make_cuio_host_pinned_pool()
+  {
+    using host_pooled_mr = rmm::mr::pool_memory_resource<rmm::mr::pinned_memory_resource>;
+    static std::shared_ptr<host_pooled_mr> mr = std::make_shared<host_pooled_mr>(
+      std::make_shared<rmm::mr::pinned_memory_resource>().get(), size_t{1} * 1024 * 1024 * 1024);
```

Comment:
I can't even believe this works. Please use the new `rmm::mr::pinned_host_memory_resource`, not the old (`host_memory_resource`) one.